cmd/cluster/transferowner.go

```diff
@@ -0,0 +1,430 @@
+package cluster
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/openshift-online/ocm-cli/pkg/arguments"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift/osdctl/internal/utils/globalflags"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+// transferOwnerOptions defines the struct for running transferOwner command
+type transferOwnerOptions struct {
+	output            string
+	clusterID         string
+	newOwnerName      string
+	newOrganizationId string
+	oldOwnerName      string
+	oldOrganizationId string
+	dryrun            bool
+
+	genericclioptions.IOStreams
+	GlobalOptions *globalflags.GlobalOptions
+}
+
+func newCmdTransferOwner(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags, globalOpts *globalflags.GlobalOptions) *cobra.Command {
+	ops := newTransferOwnerOptions(streams, flags, globalOpts)
+	transferOwnerCmd := &cobra.Command{
+		Use:               "transferOwner",
+		Short:             "Transfer cluster ownership to a new user (to be done by Region Lead)",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.complete(cmd, args))
+			cmdutil.CheckErr(ops.run())
+		},
+	}
+	// can we get cluster-id from some context maybe?
+	transferOwnerCmd.Flags().StringVarP(&ops.clusterID, "cluster-id", "C", "", "The Internal Cluster ID/External Cluster ID/ Cluster Name")
+	transferOwnerCmd.Flags().StringVar(&ops.newOwnerName, "new-owner", ops.newOwnerName, "The new owners username to transfer the cluster to")
+	transferOwnerCmd.Flags().StringVar(&ops.newOrganizationId, "new-organization-id", ops.newOrganizationId, "Organization of the new owner")
```

Comment:
The organization ids could be taken from the account resources and are not strictly needed as input. 

The SOP says organization ids will be provided in the ticket so I think for safeguards it might be a good idea to require them.